### ISSUE TITLE
0°C = 273.15 K not 273.16 K

### ISSUE THREAD

**Issue by ghost** [bensaboff/CPP-Units-Conversion#1]
TL;DR: Change 273.16 in  `TemperatureType.h` to 273.15

In `TemperatureType.h`, to convert between Celsius and K, a difference of 273.16 (K) is used, but the real difference is  273.15 (K) and not  273.16 (K).  273.16 K is the triple point of water which was used for the definition of Kelvin before the Boltzmann constant was used, which hasn't directly anything to do with the difference of °C and K.

The reason this values are so close together is, that 0 °C was once defined as the freezing point of water at 101.325 kPa, which has a very close temperature to the triple point of water ( 0.01 °C at about 612 Pa).

You can get the value from the SI brochure [here](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9.pdf)  or [here](https://www.bipm.org/en/publications/si-brochure) under 2.3.1 Base units - The kelvin (you can search for 273 to find it fast)
The linked [PDF](https://www.weather.gov/media/epz/wxcalc/tempConvert.pdf) in `TemperatureType.h` uses the wrong value of 273.16 K instead of the correct 273.15 K 

And 459.69 to convert Rankine to Fahrenheit (and later to degree Celsius) and vice versa is also wrong, it should be 459.67 instead.
